static/data/organisations.yml

```diff
@@ -0,0 +1,251 @@
+- name: '2hire'
```

Comment:
@Eomm am just curious...what could be the upside of having this data in `.yml` versus `.json`

Reply:
i've seen a script then generates the `.json` files....what could could be the intent in first having `yml` then run a script to generate `json`

Reply:
In this way we don't need to update pages/documentation across the organization - it is just another piece of the old website legacy that we could remove after the migration